gridTools.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 from matplotlib import pyplot as plt
 from scipy.interpolate import griddata
+import makenc
```

Comment:
This throws a dependency error when importing in a project that is not in the cmtb folder.  I moved to the function to remove the entire modules dependency but ideally we remove this dependency entirely by taking the netCDF writing out of the makeTimeMeanBackgroundBathy function and have it return the input to a write function wherever makeTimeMeanBackgroundBathy is used.  

Thoughts?

Reply:
makenc as a submodule?  especially if we start using it across projects - like a general nc creation library?

Reply:
fixed.